static/panes/tree.ts

```diff
@@ -124,16 +124,19 @@ export class Tree {
         this.initCallbacks();
         this.onSettingsChange(this.settings);
 
-        this.selectize = new TomSelect(this.languageBtn, {
+        this.selectize = new TomSelect(this.languageBtn[0] as HTMLSelectElement, {
             sortField: 'name',
             valueField: 'id',
             labelField: 'name',
             searchField: ['name'],
-            options: _.map(usableLanguages, _.identity),
+            options: usableLanguages,
             items: [this.multifileService.getLanguageId()],
             dropdownParent: 'body',
             plugins: ['input_autogrow'],
-            onChange: this.onLanguageChange.bind(this),
+            onChange: (e: any) => {
+                // TomSelect says it's "Event", but we get string?
```

Comment:
I don't know who's wrong here, as the onChange callback seems to be misstyped

Reply:
This issue has popped up in a couples TS conversions, for an HTMLInputElement the it is typed as a string
https://github.com/compiler-explorer/compiler-explorer/blob/cc95e37d0a1b64701a7cd1d2044f4abbd084103b/static/panes/cfg-view.ts#L163-L189

Not sure why it's not for the select element

Reply:
Tried in [b8d8437](https://github.com/compiler-explorer/compiler-explorer/pull/3577/commits/b8d8437c1d124558b5b7a823d303967c568be648), but got a tsc error saying that it was an Event so I reverted it. Weird